src/rpms.sh

```diff
@@ -593,13 +597,164 @@ satisfied or number of unsatisfied requirements.
 =cut
 
 rlCheckMakefileRequires() {
-  local req
-  rlGetYAMLdeps 'require|recommend' req || \
-  req=( $(rlGetMakefileRequires) ) || return 255
-  rlCheckRequirements "${req[@]}"
+  rlLogWarning "$FUNCNAME: considering FMF dependencies through metadata.yaml will be removed in near future"
+  rlLogWarning "$FUNCNAME:   use rlCheckDependencies or tandem rlCheckRequired / rlCheckRecommended instead"
+  local res=0 req=()
+  rlGetRecommended req
+  if [[ -n "$req" ]]; then
+    rlLogInfo "recommended:"
+    rlCheckRequirements "${req[@]}"
+  fi
+  rlGetRequired req
+  let res=$?
```

Comment:
Is there a reason for using `let` for assignment?

Reply:
No specific reason. It originally was `let res++` or something like that.